kolibri/content/api.py

```diff
@@ -4,221 +4,173 @@
 """
 from functools import wraps
 
-from django.core.files import File as DjFile
 from django.db.models import Q
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
+from .constants import content_kinds
 
 """ContentDB API methods"""
 
 def can_get_content_with_id(func):
     """
-    decorator function for returning ContentMetadata object when giving content id.
+    decorator function for returning ContentNode object when giving content id.
```

Comment:
Maybe something like `transform_content_id_to_object` would make more sense. And then update the docs like so:

```
For functions with this decorator, we check if the `content_id` is an integer. If so, we get the `ContentNode` corresponding to that id. If it's already a `ContentNode` object, just return the object itself.
```

However, can you also add an explanation as to how the function itself works, maybe as comments? It looks a bit complicated.